app/globals.css

```diff
@@ -117,3 +117,49 @@ body {
   height: 31.25rem;
   gap: 7.5rem;
 }
+
+.technology-text {
+  line-height: normal;
+}
+
+.culture-box {
+  width: 100vw;
+  padding-top: 17rem;
+  padding-left: 10rem;
+  position: relative;
+}
+
+.culture-title {
+  font-size: 5rem;
+  line-height: normal;
+  font-weight: 800;
+  padding-right: 3rem;
+}
+
+.box {
+  display: flex;
+  background-color: rgb(64, 64, 64);
+  width: 40rem;
+  height: 37rem;
+  top: 22rem;
+  right: 0;
+  border-radius: 0rem;
+  position: absolute;
+}
+
+.culture-text {
+  text-shadow: 0.1em 0.1em 0.2em rgb(49, 48, 48)
```

Comment:
In this way, the shadow doesn't work. I tested it, and it didn't work.

![image](https://github.com/grupotesseract/grupotesseract.com.br/assets/93331277/e0075632-c190-41f6-ae93-adf558856ea0)
![image](https://github.com/grupotesseract/grupotesseract.com.br/assets/93331277/4a014c4d-b310-4e10-82cc-951058b20462)


You have to use it as it is with the values ​​of the x and y axes.

![image](https://github.com/grupotesseract/grupotesseract.com.br/assets/93331277/2ab15458-a606-422b-a31d-b73ffef3f5ca)

![image](https://github.com/grupotesseract/grupotesseract.com.br/assets/93331277/6d8bf68a-e19c-42fb-bcad-6ace663f0f8b)

Reply:
And how to do fix the gray box so that it stays at the specific point relative to the text and fixed to the right.

Reply:
Nice. What I was saying are exactly this, just an information to you change the color part of text-shadow to rgba() mode, keeping all other values. Sorry if I wasn't clear in my information. Nice work.

Reply:
> And how to do fix the gray box so that it stays at the specific point relative to the text and fixed to the right.

For this, you would need to make the box adjust itself using the screen width (something like `30vw` or `40vw`). For the text, you would need to put the text inside this box, without position absolute or float, to keep it inside the boundaries of the box.

Did I answer your question? If not, maybe I may not have understood correctly.